pkg/cmd/cli/cmd/deploy.go

```diff
@@ -211,6 +211,9 @@ func (o DeployOptions) RunDeploy() error {
 // deploy launches a new deployment unless there's already a deployment
 // process in progress for config.
 func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer) error {
+	if config.Spec.Paused {
+		return fmt.Errorf("cannot deploy a paused deploymentconfig")
```

Comment:
When describing to end users we use "deployment config" or "deployment
configuration".  When we present commands or other semi-actionable output
(oc status) we use "dc/*".  We shouldn't use "deploymentconfig" ever.

On Wed, Jun 1, 2016 at 2:57 PM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/deploy.go
> <https://github.com/openshift/origin/pull/9086#discussion_r65421656>:
>
> > @@ -211,6 +211,9 @@ func (o DeployOptions) RunDeploy() error {
> >  // deploy launches a new deployment unless there's already a deployment
> >  // process in progress for config.
> >  func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer) error {
> > +	if config.Spec.Paused {
> > +		return fmt.Errorf("cannot deploy a paused deploymentconfig")
>
> I think "deploymentconfig" is fine as we already return it for anything
> cli-related. Try creating, or editing a dc and you will get
> "deploymentconfig" back. Cli users that pause their configs are probably
> familiar with our api too (/deploymentconfigs/namespace/name)
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9086/files/bbb3b59832e3980da80e07404d17b43aad07d6a4#r65421656>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p7oRPMSaKRs-ctOovTCykzZVnlWWks5qHdYxgaJpZM4Ip6MH>
> .
>